src/js/listbox.js

```diff
@@ -0,0 +1,110 @@
+//@deno-types=./19.ts
+import { $, $$, css, dispatch, makelogger, on, tag } from "./19.js"
+import { validate } from "./validate.js"
+import { FormElementMixin} from "./forms.js"
+import { FocusGroupMixin } from "./focusgroup.js"
+import { TypeAheadMixin } from "./typeahead.js"
+import { SelectableMixin } from "./selectable.js"
+import { MultiSelectMixin } from "./multiselect.js"
+
+const ilog = makelogger("listbox")
+
+export const listbox = tag(
+  "aria-listbox",
+  {
+    internals: { role: "listbox", ariaOrientation: "vertical", ariaMultiSelectable: "false" },
+    mixins: [
+      FormElementMixin,
+      FocusGroupMixin,
+      TypeAheadMixin,
+      MultiSelectMixin,
```

Comment:
Should we allow `MultiSelectMixin` to extend `TypeAheadMixin` (which would extend `FocusGroupMixin`)?